pulumi-azure-functions/README.md

```diff
@@ -0,0 +1,12 @@
+# pulumi-azure-functions
+
+This example is explained in depth in the blog article [Deploying Azure Functions with Pulumi and Zero](https://www.tryzero.com/blog/deploying-azure-functions-with-pulumi-and-zero).
+
+## Useful commands
```

Comment:
@srmagura can we also have deployment instructions? Or maybe a ref to official docs?